### ISSUE TITLE
node/no-missing-import false positive on chalk v5

### ISSUE THREAD

**Issue by coderaiser** [mysticatea/eslint-plugin-node#314]
Looks like the thing is `chalk v5` uses [`export`](https://github.com/chalk/chalk/blob/v5.0.0/package.json#L9) instead of `main`.

**Comment by regseb** [comment]
There is the same problem with the [_import/no-unresolved_](https://github.com/import-js/eslint-plugin-import/issues/2132) rule (_eslint-plugin-import_):

```
/home/regseb/testcase/index.js
  1:19  error  Unable to resolve path to module 'chalk'  import/no-unresolved
  1:19  error  "chalk" is not found                      node/no-missing-import
```

**Comment by hildjj** [comment]
[Ava](https://github.com/avajs/ava) is also having this [issue](https://github.com/avajs/ava/discussions/2866#discussioncomment-4691483).

**Comment by hildjj** [comment]
This is related to [this issue](https://github.com/browserify/resolve/pull/224) in resolve.

**Comment by CvBlixen** [comment]
It seems [got](https://github.com/sindresorhus/got) also has this issue.

At least for me. 🤔 

<img width="510" alt="image" src="https://user-images.githubusercontent.com/3101255/216765250-36ba7daf-97ae-4fca-b5b8-3122d4c18bd4.png">

For now I will just ignore linting that import. (See below.) No biggie, for now. 😌

Part of my `.eslintrc.json`:

```json
{
  "rules": {
    "node/no-missing-import": ["error", { "allowModules": ["got"] }],
    "import/no-unresolved": ["error", { "ignore": ["got"] }]
  }
}
```

Note that `node/no-missing-import` also seems to be affected ... 🤔

**Comment by voxpelli** [comment]
[`eslint-plugin-n`](https://github.com/eslint-community/eslint-plugin-n) is the maintained version of this module.

We switched to it in eg. [`eslint-config-standard`](https://github.com/standard/eslint-config-standard) / [`standard`](https://github.com/standard/standard) and it is maintained by me and other members of the official [ESLint community](https://github.com/eslint-community) organization.

Try that module out and if it isn't fixed there, then open a new issue in that project.
